ksonnet-gen/kubeversion/data.go

```diff
@@ -68,6 +68,71 @@ var versions = map[string]versionData{
 			"APIVersions":                      "apiVersions",
 			"ServerAddressByClientCIDR":        "serverAddressByClientCidr",
 		},
+		constructorSpecs: map[string][]CustomConstructorSpec{
+			"io.k8s.kubernetes.pkg.api.v1.Container": []CustomConstructorSpec{
+				newConstructor("new", newParam("name"), newParam("image")),
```

Comment:
It definitely alleviates my worry about a typo. The other benefit is that it might also start a pattern for pulling out other constants that we may expose in k.libsonnet (like "ClusterIP" or "LoadBalancer").